includes/payments/actions.php

```diff
@@ -253,3 +253,29 @@ function give_refresh_thismonth_stat_transients( $payment_ID ) {
 }
 
 add_action( 'save_post_give_payment', 'give_refresh_thismonth_stat_transients' );
+
+/*
+ * Add meta in payment that store page id and page url.
+ *
+ * Will add/update when user add click on the checkout page.
+ * The status of the donation doest not matter as it get change when user had made the payment successfully.
+ *
+ * @since 1.8.13
+ *
+ * @param {integer} $payment_id Payment id for which the meta value should be updated.
+ */
+function give_payment_save_page_data( $payment_id ) {
```

Comment:
@DevinWalker 
Why I did this because when I am adding Donations from importer or from Give-Manual-Donations then I don't need this functionality to run

So I can use remove_action function that will help me out to remove this action, so it won't run at the time of Importing or adding from Give-Manual-Donations  add-on.